rs/sns/testing/README.md

```diff
@@ -44,14 +43,183 @@ To run the scenario on the local PocketIC instance:
-     --test-canister-id "$(dfx canister --network http://127.0.0.1:8080 id test)"
+   bazel run //rs/sns/testing:sns-testing -- --network http://127.0.0.1:8080 run-basic-scenario \
+      --dev-identity sns-testing --nns-neuron-id 1 \
+      --test-canister-id "$(dfx canister --network http://127.0.0.1:8080 id test)"
    ```
 
 To start the scenario from scratch, you'll need to stop the running `pocket-ic-server` instance and
 remove `$PWD/sns-testing` and `$PWD/.dfx` directories before doing the steps mentioned above.
 
+## Deploying user-provided SNS
+
+### Bootstrap the network
+
+1) Launch PocketIC server:
+   ```
+   bazel run //rs/pocket_ic_server:pocket-ic-server -- --ttl 6000 --port 8888
+   ```
+
+2) Bootstrap the NNS on the launched PocketIC instance:
+   ```
+   bazel run //rs/sns/testing:sns-testing-init -- --server-url "http://127.0.0.1:8888" --state-dir "$PWD/sns-testing" --dev-identity sns-testing --deciding-nns-neuron-id 1
+   ```
+
+Once these steps are completed, the PocketIC will expose the IC network HTTP endpoint on "http://127.0.0.1:8080".
+`sns-testing` identity will be added as a hotkey to the NNS neuron with the majority voting power, so all NNS proposals made
+by `sns-testing` will be automatically accepted.
+
+Additionally, `nns-init` will output the ID of the NNS neuron that should be used to create NNS proposals.
+```
+...
+Use the following Neuron ID for further testing: 1
+```
+
+### Create the new SNS
+
+Now you can create a new SNS via NNS proposal.
+
+The suggested way to do this is to use `dfx sns propose` command. For more information please refer to the [documentation](https://internetcomputer.org/docs/building-apps/governing-apps/launching/launch-steps-1proposal#3-submit-nns-proposal-to-create-sns).
+
+Make sure to use `sns-testing` identity when creating the proposal.
+
+<details>
+<summary>Sample SNS proposal creation workflow</summary>
+<br>
+
+The example will use `//rs/sns/testing:sns_testing_canister` canister as SNS-controlled canister and will base on the [init YAML file from SNS CLI](../cli/test_sns_init_v2.yaml).
+
+**Make sure to use `start_time: null` in the swap parameters to ensure that the swap starts right away after the NNS proposal is executed.**
+
+1) Build and deploy `test` canister:
+   ```
+   bazel build //rs/sns/testing:sns_testing_canister
+   # 'r7inp-6aaaa-aaaaa-aaabq-cai' is the NNS Root canister
+   dfx canister --network http://127.0.0.1:8080 --identity sns-testing create test --controller r7inp-6aaaa-aaaaa-aaabq-cai --controller sns-testing --no-wallet
+   dfx canister --network http://127.0.0.1:8080 --identity sns-testing install test --wasm "$(bazel info bazel-bin)/rs/sns/testing/sns_testing_canister.wasm.gz"
+   ```
+
+2) Adjust init YAML file (you will need [`yq`](https://github.com/mikefarah/yq) to be installed to do this):
+   ```
+   yq -i ".dapp_canisters |= [\""$(dfx canister --network http://127.0.0.1:8080 id test)"\"]" ../cli/test_sns_init_v2.yaml
```

Comment:
I would not work with the yaml file from another crate, as it adds complexity to the instructions. 

Let's just copy the `../cli/test_sns_init_v2.yaml` file in the beginning of the instructions: 

```
cp ../cli/test_sns_init_v2.yaml sns_init.yaml
```

, explaining that the user may have their own custom config (with some recommendations for testing).

And then the instructions be written against a local file.